diff --git a/src/Makefile.am b/src/Makefile.am
index 7cceaaf..6724042 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -79,7 +79,7 @@ libsox_la_SOURCES += \
 	remix.c repeat.c reverb.c reverse.c silence.c sinc.c skeleff.c \
 	speed.c splice.c stat.c stats.c stretch.c swap.c \
 	synth.c tempo.c tremolo.c trim.c upsample.c vad.c vol.c \
-	ignore-warning.h
+	ignore-warning.h limiter.c
 if HAVE_PNG
     libsox_la_SOURCES += spectrogram.c
 endif
diff --git a/src/effects.h b/src/effects.h
index 450a5c2..c97d5a4 100644
--- a/src/effects.h
+++ b/src/effects.h
@@ -48,6 +48,7 @@
 #ifdef HAVE_LADSPA_H
   EFFECT(ladspa)
 #endif
+  EFFECT(limiter)
   EFFECT(loudness)
   EFFECT(lowpass)
   EFFECT(mcompand)
